### PR TITLE
Added all components

### PR DESCRIPTION
**Pull request by nishasen**
Added all the compoents need for pages

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
The code looks clean and good, No Changes suggested

File: social/src/Components/SuggestCard/SuggestCard.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    function debounce (cb, delay) {
+      let timeout;
+      return (...args) => {
+        clearTimeout(timeout);
+        timeout = setTimeout(()=>{
+          cb(...args)
+        }, delay)
+      }
+    }
```
Comment (dev-enforced) [inline social/src/Components/SuggestCard/SuggestCard.jsx:44]: This is something new to learn out!! Could you attach some resource about its working @nishasen ?

File: social/src/Components/BottomNav/BottomNav.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+const BottomNav = () => {
+    let navigate = useNavigate();
+    const { theme } = useSelector(state => state.theme);
+    const getActiveStyles = ({isActive}) => 
+        isActive ? {
+            color: "white",
+            borderRadius: "50%",
+            backgroundColor: "var(--primary)"
+        } : {}
```
Comment (dev-enforced) [inline social/src/Components/BottomNav/BottomNav.jsx:21]: Can we extract this function out as a utility. Would be easy to understand and apply changes if required!!

**Review by dev-enforced — APPROVED** [review]
The code structure looks really good @nishasen !! Hope to see the Live Preview of Social Media!!
Had some queries and suggestions mentioned below which you can answer . One more suggestion while refactoring you can take the components from Material UI and use barrel exports to avoid cluttering in your code!!

File: social/src/Components/SuggestCard/SuggestCard.jsx (hunk 1 above)
Comment (nishasen) [reply social/src/Components/SuggestCard/SuggestCard.jsx:44]: Yes sure will send you

File: social/src/Components/BottomNav/BottomNav.jsx (hunk 2 above)
Comment (nishasen) [reply social/src/Components/BottomNav/BottomNav.jsx:21]: Okay will do the changes

**Pull request merged**